src/data_operations.py

```
@@ -5,35 +5,59 @@
+    combined_data = combined_data.ffill()
+    save_data(combined_data, f"{data_type}_combined", dataPath, SAVE_AS_PICKLE)
     if PUSH_TO_GCP:
-        upload_combined_to_gcp(sofr_data, "sofr/sofr_data", bucket, SAVE_AS_PICKLE)
-    return audit_data
+        upload_to_gcp(combined_data, f"{data_type}_combined", bucket, SAVE_AS_PICKLE)
+    
+    return [{
+        "Series Name": f"{data_type}_Combined",
+        "Last Date": combined_data[col_date].max(),
+        "Last Value": ", ".join([f"{series}: {combined_data[series].iloc[-1]:.2f}" for series in data_info['data_ref']]),
+        "Last Request Datetime": datetime.now().strftime("%Y-%m-%d %H:%M:%S")
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)